src/services/passwordGeneration.service.ts

```diff
@@ -171,12 +172,23 @@ export class PasswordGenerationService implements PasswordGenerationServiceAbstr
             o.wordSeparator = ' ';
         }
 
+        if (o.commonRequirements == null) {
+            o.addCommonRequirements = false;
+        }
+
         const listLength = EEFLongWordList.length - 1;
-        const wordList = new Array(o.numWords);
+        let wordList = new Array(o.numWords);
         for (let i = 0; i < o.numWords; i++) {
             const wordIndex = await this.cryptoService.randomNumber(0, listLength);
             wordList[i] = EEFLongWordList[wordIndex];
         }
```

Comment:
I would do capitalize as part of this existing loop. That way the words do not have to be iterated again.